tslint.json

```diff
@@ -66,12 +66,7 @@
             "check-open-brace",
             "check-whitespace"
         ],
-        "ordered-imports": [
-            true,
-            {
-                "named-imports-order": "lowercase-last"
-            }
-        ],
+        "ordered-imports": false,
```

Comment:
I disabled because tslint was giving me a really hard time about the ordering of things which to me looked like they were perfectly fine and in alphabetic order. It seemed a bit irrelevant to spend my time on arranging them as I was hoping to get my changes out.